src/index.js

```diff
@@ -16,6 +17,8 @@ async function main(args) {
 
   if (interactive) {
     handler = handlers.manual;
+  } else if (args.module && fs.existsSync('yarn.lock')) {
+    handler = handlers.yarn;
   } else if (args.module) {
     handler = handlers.npm;
   }
```

Comment:
```suggestion
  } else if (args.module) {
    if (fs.existsSync('yarn.lock')) {
      handler = handlers.yarn;
    } else {
      handler = handlers.npm;
    }
  }
```

Reply:
Actually, I prefer to check twice, and prevent the double nesting. Keep the structure flat.